core/controllers/library.py

```diff
@@ -292,15 +355,33 @@ class ExplorationSummariesHandler(base.BaseHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'stringified_exp_ids': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
```

Comment:
Ditto as below.

Reply:
SVS architecture depend entirely in schema_utils for normalizing the argumnets.
For handling the case as mentioned by you we have to update schema_utils which is not so much efficient approach.
Looking forward for your point.

Reply:
Ditto as above.